src/execution/__tests__/flattenAsyncIterable-test.ts

```diff
@@ -143,15 +120,20 @@ describe('flattenAsyncIterable', () => {
     expect(caughtError).to.equal('ouch');
   });
   /* c8 ignore start */
```

Comment:
Can you remove the c8 ignore wrapping this test?

Reply:
Done